interfaces/builtin/block_devices.go

```diff
@@ -95,6 +95,10 @@ capability sys_admin,
 # Devices for various controllers used with ioctl()
 /dev/mpt2ctl{,_wd} rw,
 /dev/megaraid_sas_ioctl_node rw,
+
+# Allow /sys/block/sdX/device/state to be accessible to accept or reject the request from given the path.
+# To take the path offline will cause any subsequent access to fail immediately, vice versa.
+/sys/devices/**/host*/**/state rw,
```

Comment:
On my machine for example /sys/block/sda/device/state is a link to :

```
$ readlink -f /sys/block/sda/device/state
/sys/devices/pci0000:00/0000:00:01.1/0000:01:00.1/ata1/host0/target0:0:0/0:0:0:0/state
```

It would be great if we could somehow limit this more, but it's unclear to me how... Maybe we could start with something like

```suggestion
/sys/devices/**/ata*/host*/**/state rw,
```

to limit to sata devices (or whatever devices are deemed necessary / relevant?

Reply:
right, it could be HW dependent, from my device, it's arm64 and sdcard is connected to usb storage:
```
/sys/devices/platform/e0000000.n_ahb/e001f000.ehci/usb1/1-1/1-1.1/1-1.1:1.0/host0/target0:0:0/0:0:0:0/state
```
but it's linked to `/dev/[sda|sdb]` in this case. I agree to limit the path to be more strict, but maybe you can suggest how we make it better.

Reply:
Hmm, I think in this case it's too difficult to limit it meaningfully, and this PR does already give write access to the raw devices so I think this is fine as-is

Reply:
OK, sounds great to proceed this. Thanks for reviewing @anonymouse64